repology/parsers/parsers/debian.py

```diff
@@ -131,6 +131,10 @@ def _extra_handling(self, pkg: PackageMaker, pkgdata: dict[str, str]) -> None:
             pkg.add_name(pkgdata['SourceName'], NameType.OPENWRT_SOURCENAME)
         pkg.set_arch(pkgdata['Architecture'])
         pkg.set_extra_field('path', '/'.join(pkgpath[2:]))
+        pkg.add_homepages(pkgdata.get('URL'))
```

Comment:
You sure? I don't see any `URL` fields in either subrepo for 23.05. `CPE-ID`'s are there though.

Reply:
Right, it was only added in https://downloads.openwrt.org/releases/24.10.0/targets/apm821xx/nand/packages/Packages.manifest, sorry